.travis.yml

```
@@ -27,6 +27,11 @@ jobs:
     install: pip install -r requirements_dev.txt
     script: black --check --diff .
     after_success: skip
+  - stage: codestyle
```

```suggestion
  - stage: mypy
```
Is it better to use a different stage name, or is it ok to have duplicates?